src/g_canvas.c

```diff
@@ -2045,19 +2045,19 @@ void pd_doloadbang(void);
 
     /* evaluate a file, which is expected to create a patch, and perform
     post-evaluation cleanup and loadbang */
-t_pd *glob_evalfile(t_pd *ignore, t_symbol *name, t_symbol *dir)
+t_pd* glob_evalfile(t_pd* ignore, t_symbol* name, t_symbol* dir)
```

Comment:
no.
please keep indentation, spacing,... as they are and adjust your coding style to that of Pd.
if you found a coding-style inconsistency, you might want to fix it - but in a *separate* commit.